samples/authorization/csm_authorization_proxy_server_v200-alpha.yaml

```diff
@@ -93,3 +98,12 @@ data:
   csm-config-params.yaml: |
     CONCURRENT_POWERFLEX_REQUESTS: 10
     LOG_LEVEL: debug
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: redis-csm-cm
```

Comment:
The configMap will be updated dynamically hence kept here.

Reply:
It will be dynamically updated by the user?

Reply:
@shaynafinocchiaro redis.conf is needed for redis to start. In this case, we create an empty redis.conf and mount them as a CM in the pod for redis to start. When first pod comes up, it does nothing. When the 2+ pod comes up, it updates the pod dynamically to capture who is the capture that they are replicating from. That way the replication is established between the replica and the master. Users do not need to update the conf file, we will update them during deployment. Does this help clarify?

Reply:
@bharathsreekanth yes, thank you. If we are updating this (not the user), then I don't think this should be in the sample CR. It would be better suited in the deployment.yaml or another manifest. That's all I was trying to suggest. Sorry for any confusion on my part.

Reply:
Thanks @shaynafinocchiaro . I suppose we could add this in another manifest. @harshitap26 can you take care of that please?

Reply:
Will it be fine if I add it in deployment.yaml @bharathsreekanth @shaynafinocchiaro

Reply:
Yes I'm good with that.

Reply:
@bharathsreekanth @shaynafinocchiaro Adding the ConfigMap in policies.yaml since all the other configMaps are also defined there. I hope that is fine ?

Reply:
Done